packages/main/src/Popup.js

```diff
@@ -77,7 +77,7 @@ const metadata = {
 		},
 
 		/**
-		 * @private
+		 * @protected
```

Comment:
It is "protected", but the reason we left it "private" is because now it will start to appear in the API reference together with the public properties. We don't have a good way to display the protected properties and that's why for now I would keep it private. If you want you can leave a comment that it is used by the derived classes.

Reply:
ok, done